index.html

```diff
@@ -33,20 +33,285 @@ <h2 class="title2">LET’S CONNECT</h2>
+            <ul class="info">
+                <li class="conopy-li">
+                    <h3 class="canopy">CANOPY</h3>
+                </li>
+                <li class="couter-image">
+                        <img src="./images/Counter.png" alt="">
+                </li>
+                <li class="back-end-li">
+                    <h3 class="back-end">Back End Dev</h3>
+                </li>
+                <li class="couter-image">
+                        <img src="./images/Counter.png" alt="">
+                </li>
+                <li>
+                    <h3 class="year">2015</h3>
+                </li>
+            </ul> 
+        <p class="info-about">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="info-2">
+                <li>
+                    <button class="button-1">html</button>
+                        
+                </li>
+                <li>
+                    <button class="button-2">css</button>
+                        
+                    
+                </li>
+                <li>
+                    <button class="button-3">Javascript</button>
+                        
+                    
+                </li>
+            </ul>
+                <button class="button-4">See Project</button>
+          
+        </div>
+        
+        </div>
+    </section>
+    <section>
+        <div class="container-1">
+            <a href="#" class="project-pics"><img class="project-pics" src="./images/Snapshoot Portfolio (3).png" alt=""></a>
+           <div class="flow">
+            <h3 class="project-title-1">Multi-Post <br> Stories</h3>
+            <ul class="info">
+                <li class="conopy-li">
+                    <h3 class="canopy">CANOPY</h3>
+                </li>
+                <li class="couter-image">
+                        <img src="./images/Counter.png" alt="">
+                </li>
+                <li class="back-end-li">
+                    <h3 class="back-end">Back End Dev</h3>
+                </li>
+                <li class="couter-image">
+                        <img src="./images/Counter.png" alt="">
+                </li>
+                <li>
+                    <h3 class="year">2015</h3>
+                </li>
+            </ul> 
+        <p class="info-about">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="info-2">
+                <li>
+                    <button class="button-1">html</button>
+                        
+                </li>
+                <li>
+                    <button class="button-2">css</button>
+                        
+                    
+                </li>
+                <li>
+                    <button class="button-3">Javascript</button>
+                        
+                    
+                </li>
+            </ul>
+                <button class="button-4">See Project</button>
```

Comment:
I suggest you Implement the button interactions (enable, hover, etc.).
Its a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) for this milestone
<img width="1660" alt="Screenshot 2022-10-29 at 10 36 44" src="https://user-images.githubusercontent.com/48546992/198821864-9ea558b3-7c74-49e1-be05-f2defcf3f2fa.png">